src/js/modules/accounts/reducer.js

```diff
@@ -0,0 +1,190 @@
+import {
+  SIGN_IN_PENDING,
+  SIGN_IN_FULFILLED,
+  SIGN_IN_REJECTED,
+  SIGN_UP_FULFILLED,
+  SIGN_UP_REJECTED,
+  SIGN_UP_PENDING,
+  SIGN_OUT_PENDING,
+  SIGN_OUT_FULFILLED,
+  SIGN_OUT_REJECTED,
+  UPDATE_USER_FULFILLED,
+  UPDATE_USER_REJECTED,
+  UPDATE_USER_PENDING,
+  OPEN_SIGN_IN_MODAL,
+  CLOSE_SIGN_IN_MODAL,
+} from "./actionTypes";
+
+const initialState = {
+  status: {
```

Comment:
Not great naming. Status of what? Also `form` should be `formName` or `formType`

Reply:
Told you about this before. Status is supposed to be general. It holds status for the entire state.

Reply:
It's still a bad name